suites/quincy/cephfs/cephfs-basic-mds-operations.yaml

```diff
@@ -0,0 +1,132 @@
+# Tier0: cephfs build evaluation
```

Comment:
nitpick... sorry
We are already in `cephfs` folder... could we avoid the prefix to the filename? IMO, the filename could be `test-core-functionality.yaml`